### PR TITLE
Deprecate Realm.getTable(Class).

### PR DESCRIPTION
**Pull request by zaki50**
@realm/java

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * BREAKING CHANGE: DynamicRealm.distinct()/distinctAsync() and Realm.distinct()/distinctAsync() now throw IllegalArgumentException instead of UnsupportedOperationException for invalid type or unindexed field.
 * BREAKING CHANGE: All thread local change listeners are now delayed until the next Looper event instead of being triggered when committing.
 * Deprecated methods: Realm.getInstance(Context). Use Realm.getInstance(RealmConfiguration) or Realm.getDefaultInstance() instead.
+* Deprecated methods: Realm.getTable(Class) which was public because of migration. Use Realm.getSchema() or DynamicRealm.getSchema() instead.
```
Comment (cmelchior) [inline CHANGELOG.md]: because of the old migration API

**Comment by cmelchior** [comment]
Apart from small comment :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (zaki50) [inline CHANGELOG.md]: done

**Comment by beeender** [comment]
:+1:

**Pull request merged**